app/src/main/java/com/github/orkest/View/MainActivity.kt

```diff
@@ -19,5 +29,61 @@ class MainActivity : AppCompatActivity() {
             }
             NavigationBar.CreateNavigationBar(navController = rememberNavController(), Constants.CURRENT_LOGGED_USER)
         }
+
+        createNotificationChannel(this)
+        promptUserToEnableNotifications(this)
+    }
+
+    /**
+     * The notification channel is required for Android 8.0 and above
+     * To inform the user to enable the notifications for this app
+     */
+    fun createNotificationChannel(context: Context) {
+        val name = "Your Channel Name"
+        val descriptionText = "Your Channel Description"
+        val importance = NotificationManager.IMPORTANCE_DEFAULT
+        val channel = NotificationChannel(NOTIFICATION_CHANNEL_ID, name, importance).apply {
+            description = descriptionText
+        }
+        val notificationManager = context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+        notificationManager.createNotificationChannel(channel)
+    }
+
+    /**
+     * Build the notification of enabling them
+     */
+    fun promptUserToEnableNotifications(activity: Context) {
+        val notificationManager = NotificationManagerCompat.from(activity)
+        if (!notificationManager.areNotificationsEnabled()) {
```

Comment:
Adding a null checker would be nice to avoid unexpected behavior

Reply:
because of the nature of the code, notificationManager cannot be null